packages/itwin/tree-widget/src/test/TreeSelector.test.tsx

```diff
@@ -0,0 +1,75 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { expect } from "chai";
+import { fireEvent, render } from "@testing-library/react";
```

Comment:
Use `render` from `./TestUtils`. Additionally to rendering component it returns `user` for making interactions.